styles.css

```diff
@@ -0,0 +1,804 @@
+@import url('http://fonts.cdnfonts.com/css/cocogoose');
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,500;0,600;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "COCOGOOSE", sans-serif;
+}
+
+.desktop-nav-1 {
+  display: none;
+}
+
+.flex {
+  display: flex;
+}
+
+.desktop-nav-2 {
+  display: none;
+}
```

Comment:
- [ ] Excellent work implementing CSS classes for reducing repetition in CSS  DRY, and also for powerful runtime effects :partying_face::+1: 💪🏻